lib/site_template/README.txt

```diff
@@ -0,0 +1,23 @@
+===============================
+Welcome to your new Jekyll Site
+===============================
+
+This file generated by Jekyll will neither be included in the site built
+when you run `jekyll build` (or `jekyll serve`) nor will it be read by
+git if you plan to have your site set up as a git repository.
```

Comment:
I see why we wouldn't want it in the compiled site, but why ignore it from your git source tree? Most repositories nowadays have a README and I'd argue it's all the better that they do 😄